### ISSUE TITLE
Add `prefixed-maven-plugin` to the build pipeline

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#348]
With [its help](https://github.com/l3r8yJ/prefixed-maven-plugin) we can check our Java implementation lints (they should be prefixed with `Lt`):

```java
@RequirePrefix(prefix = "Lt")
public interface Lint { ...
}

final class LtUnlint implements Lint { ...
} // ✅ Valid

final class LintUnlint implements Lint { ...
}  // ❌Fails build
```

**Comment by h1alexbel** [comment]
@yegor256 WDYT?

**Comment by yegor256** [comment]
@h1alexbel it's doing something similar to this: https://github.com/objectionary/lints/blob/master/src/test/java/org/eolang/lints/LintTest.java#L38-L47 ?

**Comment by h1alexbel** [comment]
@yegor256 yes, maybe we should control it on maven side as well?

**Comment by VasilevNStas** [comment]
@h1alexbel @yegor256 I would suggest closing this one. Here is the reasoning:

## The convention is already followed

Every Java lint in the codebase already has the `Lt` prefix:

- `LtAsciiOnly`, `LtReservedName`, `LtUnlintNonExistingDefect`, `LtIncorrectUnlint`, `LtAlways`, `LtMono`, `LtDfSticky`, `LtByXsl`, `LtUnlint`, `LtTestNotVerb` — all registered in `MonoLints`/`PkMono` implement `Lint` and are named `Lt*`.

Adding `prefixed-maven-plugin` would only *enforce* something that already holds, with no current violations to catch.

## An architectural test already guards the related concern

`LintTest.ensuresLintsStayInTheirPackages` (using ArchUnit) already checks that every class implementing `Lint` is package-private, which is the same "control it on the test side" idea @yegor256 pointed at:

https://github.com/objectionary/lints/blob/master/src/test/java/org/eolang/lints/LintTest.java#L30-L47

Extending that test with an ArchUnit rule `classes().that().implement(Lint.class).should().haveSimpleNameStartingWith("Lt")` would be a one-line, dependency-free alternative that keeps all checks in one place, rather than adding a new Maven plugin.

## Plugin maintenance cost

`prefixed-maven-plugin` is a third-party plugin with its own release cycle. Wiring it in (version, `@RequirePrefix` annotation, plugin config) adds a dependency for a check that currently has zero failures.

## Proposal

If you want the guarantee, I can extend `LintTest` with an ArchUnit `haveSimpleNameStartingWith("Lt")` rule instead of the plugin — same effect, no new dependency, and it lives next to the existing ArchUnit test. Otherwise, close this as wontfix since the convention already holds everywhere.

**Comment by 0crat** [comment]
@h1alexbel Thanks for suggesting an enhancement! You've earned +12 points. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) bugs too. Your running score is [+83](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
